scripts/screeps_rl_live_dashboard.py

```
@@ -149,14 +162,64 @@ def refresh_loop() -> None:
             while not self.refresh_stop.wait(self.config.auto_refresh_seconds):
                 with self.refresh_lock:
                     try:
-                        refresh = refresh_metrics(self.config.db_path, self.config.artifact_root)
+                        refresh = refresh_metrics(
+                            self.config.db_path,
+                            self.config.artifact_root,
+                            max_files_per_root=self.config.ingest_file_limit_per_root,
+                        )
                     except Exception as error:  # pragma: no cover - defensive background loop boundary
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)